app/services/support_interface/ministerial_report_export.rb

```diff
@@ -0,0 +1,60 @@
+module SupportInterface
+  class MinisterialReportExport
+    def call
+      results = subject_status_count
+      subject_mapping(results)
+
+      subject_count = Subject::SUBJECTS.count
+
+      rows = Array.new(subject_count, nil)
+
+      subject_count.times do |n|
+        rows[n] = {
+          subject: Subject::SUBJECTS[n],
```

Comment:
I think you can just iterate directly over the values of `Subject::SUBJECTS` like this:

```
Subject::SUBJECTS.each do |subject|
  rows[n] = {
    subject: subject,
    applications: 0,
    offer_received: 0,
    accepted: 0,
    application_declined: 0,
    application_rejected: 0,
  }
end
```